Sources/MapboxNavigation/NavigationMapView+VanishingRouteLine.swift

```diff
@@ -343,18 +343,33 @@ extension NavigationMapView {
     /**
      Given a congestion level, return its associated color.
      */
-    func congestionColor(for congestionLevel: String?) -> UIColor {
-        switch congestionLevel {
-        case "low":
-            return trafficLowColor
-        case "moderate":
-            return trafficModerateColor
-        case "heavy":
-            return trafficHeavyColor
-        case "severe":
-            return trafficSevereColor
-        default:
-            return trafficUnknownColor
+    func congestionColor(for congestionLevel: String?, isMain: Bool) -> UIColor {
```

Comment:
We can use default value for `isMain = true` here as well.

Reply:
Thanks for the suggestion! I'll work on that.